src/components/common/ConfirmModal.tsx

```diff
@@ -52,10 +57,12 @@ const ConfirmModal = ({
   title,
   body,
   actionLabel,
+  actionLabelStatus = 'danger',
```

Comment:
does introducing this new prop affect existing usecases for this component (e.g. in resource management overview screens like `/manage/members`)

Reply:
No it shouldn't, I made it so that existing uses of the component default to what it was before (the `danger` status on the button)